service/core/server/src/main/scala/com/lightbend/lagom/internal/server/ServiceRouter.scala

```diff
@@ -54,11 +54,10 @@ private[lagom] abstract class ServiceRouter(httpConfiguration: HttpConfiguration
    */
   override val routes: Routes = Function.unlift { request =>
     val requestHeader = toRequestHeader(request)
-    val isWebSocket = request.headers.get(HeaderNames.UPGRADE).contains("websocket")
```

Comment:
The case changed between Netty 4.0.36 and 4.0.39 from `"websocket"` to `"WebSocket"`, but this behaviour resulted in a 404 in the case of a mismatch. Non-websocket requests are already handled by Play in [`PlayRequestHandler`](https://github.com/playframework/playframework/blob/2.5.10/framework/src/play-netty-server/src/main/scala/play/core/server/netty/PlayRequestHandler.scala#L139)